index.html

```diff
@@ -10,219 +10,268 @@
-                <p>JavaSript</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/html.png" alt="js" />
-                <p>HTML</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/css.png" alt="js" />
-                <p>CSS</p>
-                </li>
-                <li class="frameworks">
-                <p class="langusge">Frameworks</p>
-                <p class="sign">></p>
-                </li>
-                
-                <li class="frameworks">
-                <p class="langusge">Skills</p>
-                <p class="sign">></p>
-                </li>
+                    <li class="lang-title">
+                        <p class="langusge">Languages</p>
+                        <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/js.png" alt="js" />
+                        <p>JavaSript</p>
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/html.png" alt="js" />
+                        <p>HTML</p>
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/css.png" alt="js" />
+                        <p>CSS</p>
+                    </li>
+                    <li class="frameworks">
+                        <p class="langusge">Frameworks</p>
+                        <p class="sign"></p>
+                    </li>
+                    <li class="frameworks">
+                        <p class="langusge">Skills</p>
+                        <p class="sign"></p>
+                    </li>
                 </ul>
-                </div>
             </div>
-
-        </section>
-            
-            </body>
-            
-            </html>
-            
-            
\ No newline at end of file
+        </section> 
+        <section class="contact-me"> 
+            <div class="contact-background">
+                <h2>Contact me</h2>
+                <p>
+                    If you have an application you are interested in developing, a feature that you need to build or a project that needs coding. I'd love to help with
+                </p>           
+                <form action="https://formspree.io/f/xvonwnvo" method="post">
+                    <input type="text" id="fname" name="fname" placeholder="lorem Ipsum" maxlength="30" required><br>
+                    <input type="email" id="email" name="email" placeholder="loremipsum@gmail.com" required><br>
+                    <textarea name="message" id="message" maxlength="500" required>Write your message here...</textarea><br>
+                    <button class="get-in"type="submit" id="submit" name="submit" value="submit">Get In Touch</button>
```

Comment:
- [ ] Kindly fix the margins of the elements in the contact me section and like the `header` ensure it spans across the width of the screen as appropriate.
<img width="589" alt="image" src="https://user-images.githubusercontent.com/41163571/230603712-42416a0f-3456-428b-a376-414b0e05aab0.png">